<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WarehouseWindow</class>
 <widget class="QMainWindow" name="WarehouseWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>–°–∏—Å—Ç–µ–º–∞ –∫–µ—Ä—É–≤–∞–Ω–Ω—è —Å–∫–ª–∞–¥–æ–º (STL Lab 8)</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow {
    background-color: #1A1A1A;
    color: #E0E0E0;
    font-family: &quot;Segoe UI&quot;, sans-serif;
    font-size: 11pt;
}

QWidget#centralwidget {
    background-color: #1A1A1A;
    border-radius: 10px;
}

QLabel {
    color: #E8E8E8;
    font-weight: bold;
}

QLineEdit {
    background-color: #252525;
    color: #E8E8E8;
    border: 1px solid #404040;
    border-radius: 6px;
    padding: 6px 10px;
    selection-background-color: #2A5599;
}
QLineEdit:focus {
    border: 1px solid #4A7BC8;
    background-color: #202020;
}

QSpinBox, QDoubleSpinBox {
    background-color: #252525;
    color: #E8E8E8;
    border: 1px solid #404040;
    border-radius: 6px;
    padding: 6px 10px;
    selection-background-color: #2A5599;
}
QSpinBox:focus, QDoubleSpinBox:focus {
    border: 1px solid #4A7BC8;
    background-color: #202020;
}

QPushButton {
    background: qlineargradient(x1:0, y1:0, x2:1, y2:1,
                                stop:0 #1E3A8A, stop:0.5 #2563EB, stop:1 #3B82F6);
    color: #FFFFFF;
    border: none;
    border-radius: 8px;
    padding: 8px 20px;
    font-weight: bold;
    min-width: 80px;
}
QPushButton:hover {
    background: qlineargradient(x1:0, y1:0, x2:1, y2:1,
                                stop:0 #2563EB, stop:0.5 #3B82F6, stop:1 #60A5FA);
    border: 4px solid #3B82F6;
    padding: 4px 16px;
}
QPushButton:pressed {
    background: qlineargradient(x1:0, y1:0, x2:1, y2:1,
                                stop:0 #1E40AF, stop:0.5 #1D4ED8, stop:1 #2563EB);
    border: none;
    padding: 8px 20px;
}

QPushButton[class=&quot;destructive&quot;] {
    background: qlineargradient(x1:0, y1:0, x2:1, y2:1,
                                stop:0 #DC2626, stop:0.5 #EF4444, stop:1 #F87171);
    color: #FFFFFF;
    border: none;
    border-radius: 8px;
    padding: 8px 20px;
    font-weight: bold;
    min-width: 80px;
}
QPushButton[class=&quot;destructive&quot;]:hover {
    background: qlineargradient(x1:0, y1:0, x2:1, y2:1,
                                stop:0 #B91C1C, stop:0.5 #DC2626, stop:1 #EF4444);
    border: 4px solid #DC2626;
    padding: 4px 16px;
}
QPushButton[class=&quot;destructive&quot;]:pressed {
    background: qlineargradient(x1:0, y1:0, x2:1, y2:1,
                                stop:0 #991B1B, stop:0.5 #B91C1C, stop:1 #DC2626);
    border: none;
    padding: 8px 20px;
}

QPushButton[class=&quot;secondary&quot;] {
    background: qlineargradient(x1:0, y1:0, x2:1, y2:1,
                                stop:0 #3F3F46, stop:0.5 #52525B, stop:1 #71717A);
    color: #FFFFFF;
    border: none;
    border-radius: 8px;
    padding: 8px 20px;
    font-weight: bold;
    min-width: 80px;
}
QPushButton[class=&quot;secondary&quot;]:hover {
    background: qlineargradient(x1:0, y1:0, x2:1, y2:1,
                                stop:0 #27272A, stop:0.5 #3F3F46, stop:1 #52525B);
    border: 4px solid #52525B;
    padding: 4px 16px;
}
QPushButton[class=&quot;secondary&quot;]:pressed {
    background: qlineargradient(x1:0, y1:0, x2:1, y2:1,
                                stop:0 #18181B, stop:0.5 #27272A, stop:1 #3F3F46);
    border: none;
    padding: 8px 20px;
}

QTableWidget {
    background-color: #1A1A1A;
    color: #D8D8D8;
    border: 1px solid #404040;
    gridline-color: #404040;
    selection-background-color: #2A5599;
    selection-color: #FFFFFF;
    border-radius: 6px;
}
QTableWidget::item {
    background-color: #1A1A1A;
    padding: 5px;
}
QTableWidget::item:selected {
    background-color: #2A5599;
}

QHeaderView::section {
    background-color: #252525;
    color: #C8C8C8;
    font-weight: bold;
    border: 1px solid #404040;
    padding: 8px;
    text-align: center;
}
QHeaderView::section:hover {
    background-color: #303030;
}

QHeaderView {
    background-color: #1A1A1A;
}

QTableWidget QTableCornerButton::section {
    background-color: #252525;
    border: 1px solid #404040;
}

QScrollBar:vertical {
    background-color: #252525;
    width: 12px;
    margin: 0px;
    border-radius: 6px;
}
QScrollBar::handle:vertical {
    background-color: #333333;
    min-height: 20px;
    border-radius: 6px;
}
QScrollBar::handle:vertical:hover {
    background-color: #4A7BC8;
}
QScrollBar::add-line:vertical, QScrollBar::sub-line:vertical {
    height: 0px;
}
QScrollBar:horizontal {
    background-color: #252525;
    height: 12px;
    border-radius: 6px;
}
QScrollBar::handle:horizontal {
    background-color: #333333;
    min-width: 20px;
    border-radius: 6px;
}
QScrollBar::handle:horizontal:hover {
    background-color: #4A7BC8;
}
QScrollBar::add-line:horizontal, QScrollBar::sub-line:horizontal {
    width: 0px;
}

QSpinBox::up-button, QDoubleSpinBox::up-button {
    background-color: #333333;
    border: 1px solid #444444;
    border-radius: 3px;
    width: 18px;
    margin: 2px;
}
QSpinBox::up-button:hover, QDoubleSpinBox::up-button:hover {
    background-color: #4A7BC8;
}
QSpinBox::down-button, QDoubleSpinBox::down-button {
    background-color: #333333;
    border: 1px solid #444444;
    border-radius: 3px;
    width: 18px;
    margin: 2px;
}

QSpinBox::down-button:hover, QDoubleSpinBox::down-button:hover {
    background-color: #4A7BC8;
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="stlLabel">
      <property name="text">
       <string>&lt;h3&gt;üîß STL –ö–æ–Ω—Ç–µ–π–Ω–µ—Ä–∏&lt;/h3&gt;</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="stlButtons2Layout">
      <item>
       <widget class="QPushButton" name="showUniqueNamesButton">
        <property name="text">
         <string>SET (–£–Ω—ñ–∫–∞–ª—å–Ω—ñ –Ω–∞–∑–≤–∏)</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="showMultimapButton">
        <property name="text">
         <string>MULTIMAP (–î—É–±–ª—ñ–∫–∞—Ç–∏)</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="showLowStockButton">
        <property name="text">
         <string>PRIORITY_QUEUE (–ù–∏–∑—å–∫—ñ –∑–∞–ø–∞—Å–∏)</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="showHistoryButton">
        <property name="text">
         <string>STACK (–Ü—Å—Ç–æ—Ä—ñ—è –∑–º—ñ–Ω)</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Orientation::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="searchLayout">
      <item>
       <widget class="QLineEdit" name="searchLineEdit">
        <property name="placeholderText">
         <string>–í–≤–µ–¥—ñ—Ç—å –≤–∏—Ä–æ–±–Ω–∏–∫–∞</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="searchLabel">
        <property name="text">
         <string>üîç –ü–æ—à—É–∫ –∑–∞ –≤–∏—Ä–æ–±–Ω–∏–∫–æ–º:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="searchButton">
        <property name="text">
         <string>–ü–æ—à—É–∫</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="showAllButton">
        <property name="text">
         <string>–ü–æ–∫–∞–∑–∞—Ç–∏ –≤—Å—ñ</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer>
        <property name="orientation">
         <enum>Qt::Orientation::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="sortPriceButton">
        <property name="text">
         <string>–°–æ—Ä—Ç—É–≤–∞—Ç–∏ –∑–∞ —Ü—ñ–Ω–æ—é</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="sortNameButton">
        <property name="text">
         <string>–°–æ—Ä—Ç—É–≤–∞—Ç–∏ –∑–∞ –Ω–∞–∑–≤–æ—é</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QTableWidget" name="tableWidget"/>
    </item>
    <item>
     <layout class="QHBoxLayout" name="stlButtonsLayout">
      <item>
       <widget class="QPushButton" name="addSupplyButton">
        <property name="text">
         <string>QUEUE (–î–æ–¥–∞—Ç–∏ –ø–æ—Å—Ç–∞—á–∞–Ω–Ω—è)</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="showSupplyQueueButton">
        <property name="text">
         <string>–ü–æ–∫–∞–∑–∞—Ç–∏ —á–µ—Ä–≥—É</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="processSupplyButton">
        <property name="text">
         <string>–û–±—Ä–æ–±–∏—Ç–∏ –ø–æ—Å—Ç–∞—á–∞–Ω–Ω—è</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QLabel" name="statsLabel">
      <property name="styleSheet">
       <string>background-color: #252525; padding: 10px; border-radius: 5px; color: #3B82F6;</string>
      </property>
      <property name="text">
       <string>–°—Ç–∞—Ç–∏—Å—Ç–∏–∫–∞: –ó–∞–≤–∞–Ω—Ç–∞–∂–µ–Ω–Ω—è...</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
